### PR TITLE
Remove reference to Neo4j Streams

### PR DESCRIPTION
**Pull request by nvitucci**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-aura-219.surge.sh

File: modules/ROOT/pages/platform/connectors/kafka.adoc [hunk 1]
```diff
@@ -1,16 +1,13 @@
 [[connecting-kafka]]
-= Neo4j Connector for Apache Kafka (Neo4j Streams)
+= Neo4j Connector for Apache Kafka
 :description: This page describes how to connect to Aura using Kafka.
 
-Many user and customers want to integrate Kafka and other streaming solutions with Neo4j, either to ingest data into the graph from other sources or to send update events to the event log for later consumption. Neo4j Streams was developed to satisfy all these use-cases and more to come.
+Many user and customers want to integrate Kafka and other streaming solutions with Neo4j, either to ingest data into the graph from other sources or to send update events to the event log for later consumption. 
+Aura supports the use of the https://neo4j.com/labs/kafka/4.1/kafka-connect/[Kafka Connect Plugin^], which allows you to ingest data into Neo4j from Kafka topics via Cypher queries.
```
Comment (ali-ince) [inline modules/ROOT/pages/platform/connectors/kafka.adoc:6]:
```suggestion
Aura supports the use of the https://neo4j.com/docs/kafka/[Kafka Connect Neo4j Connector^], which allows you to ingest data into Neo4j from Kafka topics or send change events from Neo4j into Kafka topics.
```

File: modules/ROOT/pages/platform/connectors/kafka.adoc [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
+Connecting to Aura only requires to make a few changes to the https://neo4j.com/docs/kafka/kafka-connect/source/[source^] and https://neo4j.com/docs/kafka/kafka-connect/sink/[sink^] configuration examples:
 
 * Replace the `bolt` URI in the examples (the value of the `neo4j.server.uri` configuration parameter) with the `neo4j+s://` connection URI from the Aura instance detail page
 * Update the username and password configuration parameters as appropriate
 
-For more information check the https://neo4j.com/labs/kafka/4.1/overview/[Neo4j Streams^] page.
-
-NOTE: Aura currently only supports the Kafka Connect Plugin. Connection via the Neo4j plugin is not supported.
\ No newline at end of file
+For more information check the https://neo4j.com/docs/kafka/[Neo4j Kafka Connect^] user guide.
```
Comment (ali-ince) [inline modules/ROOT/pages/platform/connectors/kafka.adoc:13]:
```suggestion
For more information check the https://neo4j.com/docs/kafka/[Kafka Connect Neo4j Connector^] user guide.
```

**Comment by nvitucci** [comment]
Thanks for the suggestions @ali-ince!

**Review by MarkWoulfeNeo4j — APPROVED** [review]

**Pull request merged**